packages/desktop-client/src/components/transactions/SelectedTransactions.js

```diff
@@ -0,0 +1,157 @@
+import React, { useMemo } from 'react';
+import { useNavigate, useLocation } from 'react-router-dom';
+
+import { useSelectedItems } from '../../hooks/useSelected';
+import { Menu } from '../common';
+import { SelectedItemsButton } from '../table';
+
+import { isPreviewId } from './TransactionsTable';
+
+export function SelectedTransactionsButton({
```

Comment:
This component was also recently updated, you may want to re-copy it from `master`